.github/workflows/upgrade-gaia-v16-fresh-interchain.yml

```
@@ -0,0 +1,98 @@
+---
+
+name: Gaia v16 Upgrade - Fresh State Interchain
+on:
+    push:
```

Comment out the "run on push" before it goes on `main`, use these instead (we can run it more than once a day once more tests are implemented).
```suggestion
   # push:
  repository_dispatch:
    types: [new_main_build]
  schedule:
    - cron: '10 * * 1-6'
  workflow_dispatch:
```